app/models/katello/docker_manifest.rb

```diff
@@ -2,7 +2,7 @@ module Katello
   class DockerManifest < Katello::Model
     include Concerns::PulpDatabaseUnit
 
-    has_one :docker_tag, :dependent => :destroy, :class_name => "Katello::DockerTag"
+    has_many :docker_tags, :dependent => :destroy, :class_name => "Katello::DockerTag"
```

Comment:
Actually might need to address the scoped search param here https://github.com/Katello/katello/blob/master/app/models/katello/docker_manifest.rb#L11 since its one to many now...